ui/src/app/views/app/component.html

```diff
@@ -7,6 +7,7 @@
 <nav *ngIf="(user | async).isSignedIn" style="margin-top: 1rem;">
   <a routerLink="/login" routerLinkActive="active">Switch User</a>
   <a routerLink="/repository" routerLinkActive="active">Select CDR</a>
+  <a routerLink="/workspace" routerLinkActive="active">My Workspace</a>
```

Comment:
I can do that. Do we have a dummy namespace and ID I should use until we have an actual workspace, or should I just use a placeholder value?